suplemon/config.py

```diff
@@ -20,25 +20,28 @@ def __init__(self, app):
         self.config_filename = "suplemon-config.json"
         self.keymap_filename = "suplemon-keymap.json"
         self.home_dir = os.path.expanduser("~")
-        self.fpath = os.path.join(self.home_dir, ".config", "suplemon")
+        self.config_dir = os.path.join(self.home_dir, ".config", "suplemon")
 
         self.defaults = {}
         self.keymap = {}
         self.config = {}
 
+    def get_config_dir(self):
+        return self.config_dir
```

Comment:
I would say keep it simple (i.e. use `config_dir` directly). If you need a quick hack later on, Python has `getter/setter` methods:

```python
self._config_dir = os.path.join(self.home_dir, ".config", "suplemon")

@property
def config_dir(self):
  return self._config_dir
```

Aside from that, replacing the code to use whatever edits are required (e.g. `config_dir` to `user_config_dir`) should be updating the code in a handful of places, not something like 100 =/